src/themes/luminara.css

```diff
@@ -92,5 +92,12 @@ body[data-theme="luminara"] {
   .themed-button {
     background: var(--color-light-grey);
   }
+
+  .question-container, .dashboard-container {
+    /* WORKAROUND: table border radius */
+    [data-testid="TableInteractive-root"] {
+      border-radius: 6px;
+    }
```

Comment:
Is this more like a problem we should fix at the SDK level instead of patching it here?

Do we have a task for this yet?

Reply:
@WiNloSt This is applying styles so they match the Luminara theme's styles of rounded corners.

I think we have to consider if we want to expose a CSS class instead of a theme option too.

Reply:
so our tables are always without rounded corner?

Reply:
Yep, current styles is no border radius afaik